lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  --policy-indicator-highlight-colour: #{$nx-threat-color-unspecified};
```

Comment:
We use the american spelling of "color"

Reply:
> We use the american spelling of "color"

Damn these Canadian fingers of mine: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/9253b5a25ae5d96ad83f1f22af1d820ddfbacab9